### PR TITLE
MIVisionX Online Documentation Fix

### PR DESCRIPTION
**Pull request by kiritigowda**
https://gpuopen-professionalcompute-libraries.github.io/MIVisionX/ - Website has errors reading the readme.md's

This PR fixes the .md to html translations.

### CODE REVIEW

**Comment by hansely** [comment]
Looks good to me. More straightforward and concise.

File: apps/cloud_inference/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 | Inference Application Development Workflow | Sample Cloud Inference Application |
 |-------------------------|-------------------------|
-| <p align="center"><img width="80%" src="../../docs/images/block_diagram_inference_workflow.png"/></p> | <p align="center"><img width="80%" src="../../docs/images/block_diagram_inference_sample.png"/></p> |
+| <img width="80%" src="../../docs/images/block_diagram_inference_workflow.png"/> | <img width="80%" src="../../docs/images/block_diagram_inference_sample.png"/>|
```
Comment (kiritigowda) [inline apps/cloud_inference/README.md:5]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929&bid=19863286)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 30
- Added 20
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929&bid=19863286)

File: utilities/inference_generator/README.md [hunk 2]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+|--[no-]virtual-buffers    | do/don't use virtual buffers (default: ON)|
+|--[no-]generate-gdf       | do/don't generate RunVX GDF with weight/bias initialization (default: ON)|
+|--[no-]generate-vx-code   | do/don't generate OpenVX C Code with weight/bias initialization (default: ON)|
+|--output-dir <folder>     | specify output folder for weights/biases, GDF, and OpenVX C Code (default: current)|
+|--input-rgb <a> <b> <rev> | convert input from RGB image into tensor using (a*x+b) conversion: rev=(BGR?1:0)|
+|--input-u8  <a> <b>       | convert input from U8 image into tensor using (a*x+b) conversion|
+|--argmax-tensor u8|u16 k  | return argmax output with specified tensor type and top_k|
+|--argmax-image u8|u16     | return argmax output with specified image type|
+|--argmax-lut <rgbLut.txt> | argmax color table: one R G B entry per label|
+|--flags <int>             | specify custom flags (default: 0)|
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:79]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 3]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
---argmax-lut <rgbLut.txt> | argmax color table: one R G B entry per label
---flags <int>             | specify custom flags (default: 0)
+```
+
+| option | description |
+| ------ | ----------- |
+|--[no-]error-messages     | do/don't enable error messages (default: ON)|
+|--[no-]virtual-buffers    | do/don't use virtual buffers (default: ON)|
+|--[no-]generate-gdf       | do/don't generate RunVX GDF with weight/bias initialization (default: ON)|
+|--[no-]generate-vx-code   | do/don't generate OpenVX C Code with weight/bias initialization (default: ON)|
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:72]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 4]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+|--output-dir <folder>     | specify output folder for weights/biases, GDF, and OpenVX C Code (default: current)|
+|--input-rgb <a> <b> <rev> | convert input from RGB image into tensor using (a*x+b) conversion: rev=(BGR?1:0)|
+|--input-u8  <a> <b>       | convert input from U8 image into tensor using (a*x+b) conversion|
+|--argmax-tensor u8|u16 k  | return argmax output with specified tensor type and top_k|
+|--argmax-image u8|u16     | return argmax output with specified image type|
+|--argmax-lut <rgbLut.txt> | argmax color table: one R G B entry per label|
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:78]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 5]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
---argmax-image u8|u16     | return argmax output with specified image type
---argmax-lut <rgbLut.txt> | argmax color table: one R G B entry per label
---flags <int>             | specify custom flags (default: 0)
+```
+
+| option | description |
+| ------ | ----------- |
+|--[no-]error-messages     | do/don't enable error messages (default: ON)|
+|--[no-]virtual-buffers    | do/don't use virtual buffers (default: ON)|
+|--[no-]generate-gdf       | do/don't generate RunVX GDF with weight/bias initialization (default: ON)|
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:71]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 6]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
---input-u8  <a> <b>       | convert input from U8 image into tensor using (a*x+b) conversion
---argmax-tensor u8|u16 k  | return argmax output with specified tensor type and top_k
---argmax-image u8|u16     | return argmax output with specified image type
---argmax-lut <rgbLut.txt> | argmax color table: one R G B entry per label
---flags <int>             | specify custom flags (default: 0)
+```
+
+| option | description |
+| ------ | ----------- |
+|--[no-]error-messages     | do/don't enable error messages (default: ON)|
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:69]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 4 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:78]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 7]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+|--[no-]virtual-buffers    | do/don't use virtual buffers (default: ON)|
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:70]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 8]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+|--[no-]generate-gdf       | do/don't generate RunVX GDF with weight/bias initialization (default: ON)|
+|--[no-]generate-vx-code   | do/don't generate OpenVX C Code with weight/bias initialization (default: ON)|
+|--output-dir <folder>     | specify output folder for weights/biases, GDF, and OpenVX C Code (default: current)|
+|--input-rgb <a> <b> <rev> | convert input from RGB image into tensor using (a*x+b) conversion: rev=(BGR?1:0)|
+|--input-u8  <a> <b>       | convert input from U8 image into tensor using (a*x+b) conversion|
+|--argmax-tensor u8|u16 k  | return argmax output with specified tensor type and top_k|
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:76]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

Comment (kiritigowda) [inline utilities/inference_generator/README.md:76]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 5 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:71]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 6 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:69]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 7 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:70]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: apps/cloud_inference/README.md (hunk 1 above)
Comment (kiritigowda) [inline apps/cloud_inference/README.md:5]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 3 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:72]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 9]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+
+| option | description |
+| ------ | ----------- |
+|--[no-]error-messages     | do/don't enable error messages (default: ON)|
+|--[no-]virtual-buffers    | do/don't use virtual buffers (default: ON)|
+|--[no-]generate-gdf       | do/don't generate RunVX GDF with weight/bias initialization (default: ON)|
+|--[no-]generate-vx-code   | do/don't generate OpenVX C Code with weight/bias initialization (default: ON)|
+|--output-dir <folder>     | specify output folder for weights/biases, GDF, and OpenVX C Code (default: current)|
+|--input-rgb <a> <b> <rev> | convert input from RGB image into tensor using (a*x+b) conversion: rev=(BGR?1:0)|
+|--input-u8  <a> <b>       | convert input from U8 image into tensor using (a*x+b) conversion|
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:75]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 2 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:79]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 3 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:72]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 9 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:75]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 10]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+|--argmax-tensor u8|u16 k  | return argmax output with specified tensor type and top_k|
+|--argmax-image u8|u16     | return argmax output with specified image type|
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:77]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 30
- Added 19
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929&bid=19863286)

File: utilities/inference_generator/README.md (hunk 8 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 7 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 6 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 3 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 10 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 8 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 5 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 4 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 2 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 3 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 2 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 5 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 10 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 9 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

Comment (kiritigowda) [inline utilities/inference_generator/README.md]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 35
- Added 15
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929&bid=19863286)

File: utilities/inference_generator/README.md [hunk 11]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+| ------ | ----------- |
+| --[no-]error-messages     | do/don't enable error messages (default: ON) |
+| --[no-]virtual-buffers    | do/don't use virtual buffers (default: ON) |
+| --[no-]generate-gdf       | do/don't generate RunVX GDF with weight/bias initialization (default: ON) |
+| --[no-]generate-vx-code   | do/don't generate OpenVX C Code with weight/bias initialization (default: ON) |
+| --output-dir <folder>     | specify output folder for weights/biases, GDF, and OpenVX C Code (default: current) |
+| --input-rgb <a> <b> <rev> | convert input from RGB image into tensor using (a*x+b) conversion: rev=(BGR?1:0) |
+| --input-u8  <a> <b>       | convert input from U8 image into tensor using (a*x+b) conversion |
+| --argmax-tensor u8|u16 k  | return argmax output with specified tensor type and top_k |
+| --argmax-image u8|u16     | return argmax output with specified image type |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:77]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 12]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
---argmax-image u8|u16     | return argmax output with specified image type
---argmax-lut <rgbLut.txt> | argmax color table: one R G B entry per label
---flags <int>             | specify custom flags (default: 0)
+```
+
+| option | description |
+| ------ | ----------- |
+| --[no-]error-messages     | do/don't enable error messages (default: ON) |
+| --[no-]virtual-buffers    | do/don't use virtual buffers (default: ON) |
+| --[no-]generate-gdf       | do/don't generate RunVX GDF with weight/bias initialization (default: ON) |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:71]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 13]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
---input-u8  <a> <b>       | convert input from U8 image into tensor using (a*x+b) conversion
---argmax-tensor u8|u16 k  | return argmax output with specified tensor type and top_k
---argmax-image u8|u16     | return argmax output with specified image type
---argmax-lut <rgbLut.txt> | argmax color table: one R G B entry per label
---flags <int>             | specify custom flags (default: 0)
+```
+
+| option | description |
+| ------ | ----------- |
+| --[no-]error-messages     | do/don't enable error messages (default: ON) |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:69]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 14]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+| --[no-]virtual-buffers    | do/don't use virtual buffers (default: ON) |
+| --[no-]generate-gdf       | do/don't generate RunVX GDF with weight/bias initialization (default: ON) |
+| --[no-]generate-vx-code   | do/don't generate OpenVX C Code with weight/bias initialization (default: ON) |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:72]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 11 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:77]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 15]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+| --output-dir <folder>     | specify output folder for weights/biases, GDF, and OpenVX C Code (default: current) |
+| --input-rgb <a> <b> <rev> | convert input from RGB image into tensor using (a*x+b) conversion: rev=(BGR?1:0) |
+| --input-u8  <a> <b>       | convert input from U8 image into tensor using (a*x+b) conversion |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:75]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 12 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:71]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 16]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+| --argmax-tensor u8|u16 k  | return argmax output with specified tensor type and top_k |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:76]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 17]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
---argmax-tensor u8|u16 k  | return argmax output with specified tensor type and top_k
---argmax-image u8|u16     | return argmax output with specified image type
---argmax-lut <rgbLut.txt> | argmax color table: one R G B entry per label
---flags <int>             | specify custom flags (default: 0)
+```
+
+| option | description |
+| ------ | ----------- |
+| --[no-]error-messages     | do/don't enable error messages (default: ON) |
+| --[no-]virtual-buffers    | do/don't use virtual buffers (default: ON) |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:70]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 18]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+| --[no-]generate-gdf       | do/don't generate RunVX GDF with weight/bias initialization (default: ON) |
+| --[no-]generate-vx-code   | do/don't generate OpenVX C Code with weight/bias initialization (default: ON) |
+| --output-dir <folder>     | specify output folder for weights/biases, GDF, and OpenVX C Code (default: current) |
+| --input-rgb <a> <b> <rev> | convert input from RGB image into tensor using (a*x+b) conversion: rev=(BGR?1:0) |
+| --input-u8  <a> <b>       | convert input from U8 image into tensor using (a*x+b) conversion |
+| --argmax-tensor u8|u16 k  | return argmax output with specified tensor type and top_k |
+| --argmax-image u8|u16     | return argmax output with specified image type |
+| --argmax-lut <rgbLut.txt> | argmax color table: one R G B entry per label |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:78]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 16 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:76]: Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 17 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:70]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 19]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+| --flags <int>             | specify custom flags (default: 0) |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:79]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 13 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:69]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md (hunk 14 above)
Comment (kiritigowda) [inline utilities/inference_generator/README.md:72]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 35
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929&bid=19863286)

File: utilities/inference_generator/README.md [hunk 20]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+| --(no-)error-messages     | do/don't enable error messages (default: ON) |
+| --(no-)virtual-buffers    | do/don't use virtual buffers (default: ON) |
+| --(no-)generate-gdf       | do/don't generate RunVX GDF with weight/bias initialization (default: ON) |
+| --(no-)generate-vx-code   | do/don't generate OpenVX C Code with weight/bias initialization (default: ON) |
+| --output-dir <folder>     | specify output folder for weights/biases, GDF, and OpenVX C Code (default: current) |
+| --input-rgb <a> <b> <rev> | convert input from RGB image into tensor using (a*x+b) conversion: rev=(BGR?1:0) |
+| --input-u8  <a> <b>       | convert input from U8 image into tensor using (a*x+b) conversion |
+| --argmax-tensor u8/u16 k  | return argmax output with specified tensor type and top_k |
+| --argmax-image u8/u16     | return argmax output with specified image type |
+| --argmax-lut <rgbLut.txt> | argmax color table: one R G B entry per label |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:78]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 21]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+| --flags <int>             | specify custom flags (default: 0) |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:79]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 22]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+
+| option | description |
+| ------ | ----------- |
+| --(no-)error-messages     | do/don't enable error messages (default: ON) |
+| --(no-)virtual-buffers    | do/don't use virtual buffers (default: ON) |
+| --(no-)generate-gdf       | do/don't generate RunVX GDF with weight/bias initialization (default: ON) |
+| --(no-)generate-vx-code   | do/don't generate OpenVX C Code with weight/bias initialization (default: ON) |
+| --output-dir <folder>     | specify output folder for weights/biases, GDF, and OpenVX C Code (default: current) |
+| --input-rgb <a> <b> <rev> | convert input from RGB image into tensor using (a*x+b) conversion: rev=(BGR?1:0) |
+| --input-u8  <a> <b>       | convert input from U8 image into tensor using (a*x+b) conversion |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:75]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 23]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+| --argmax-tensor u8/u16 k  | return argmax output with specified tensor type and top_k |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:76]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

File: utilities/inference_generator/README.md [hunk 24]
```diff
@@ -48,40 +50,46 @@ vx_graph annCreateGraphWithInputImageWithArgmaxImageWithLut(
...
+| --argmax-image u8/u16     | return argmax output with specified image type |
```
Comment (kiritigowda) [inline utilities/inference_generator/README.md:77]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 35
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6064929&bid=19863286)

**Pull request merged**